pkg/machine/ignition.go

```diff
@@ -89,6 +90,24 @@ ExecStart=/bin/sh -c '/usr/bin/echo Ready >/dev/%s'
 [Install]
 RequiredBy=multi-user.target
 `
+	deMoby := `[Unit]
+Description=Remove moby-engine, symlink docker to podman
+# Run once for the machine
+After=systemd-machine-id-commit.service
+Before=zincati.service
+ConditionPathExists=!/var/lib/%N.stamp
+
+[Service]
+Type=oneshot
+RemainAfterExit=yes
+ExecStart=/usr/bin/rpm-ostree override remove moby-engine
+ExecStart=/bin/ln -fs /usr/bin/podman /usr/local/bin/docker
+ExecStart=/bin/touch /var/lib/%N.stamp
+ExecStart=/usr/bin/rpm-ostree ex apply-live --allow-replacement
```

Comment:
```suggestion
ExecStart=/usr/bin/rpm-ostree ex apply-live --allow-replacement
ExecStart=/bin/touch /var/lib/%N.stamp
```

Is there a good reason to create the stamp file before we're done? If there is I'm fine with it like it is. I just usually create stamp files like this once the unit has completed all other work.

Reply:
The original pattern I followed had a systemd restart so it's just a carryover with that. Since I use apply live, I abandoned that approach since it dramatically increased the first start time, and switched to apply live. With that change,  it could be on completion instead. I'll make that change.